modules/administration/pages/ptfs.adoc

```diff
@@ -0,0 +1,103 @@
+== Understanding PTF packages
+
+PTF packages are installed via a proxy package and are named `ptf-xxxxxx`.
+They will depend on the correct version of the package that is known to include the correction in the software. This type of package:
+
+* cannot be installed accidentally (i.e. zypper update will never suggest installing them),
+* cannot be removed accidentally (i.e. a newer package version will not replace the PTF one, unless the user makes it explicitly on the zypper command line),
+* is only updated when the newer version is known to address the specific issue previously solved by the PTF,
+* updates only packages already installed on the system (i.e. if a software is split into multiple packages, the PTF will replace only those currently installed on the system).
+
+The correct ID of the package will be provided by SUSE Support during the course of the support case investigation, along with instruction on how to deploy/restart the affected services.
+
+
+== Installing PTF packages
+
+
+[NOTE]
+====
+PTF packages are currently only supported for SLE 12 and SLE 15 based systems.
+Other versions or Operating Systems do not have this feature yet and the pages are not visible for them.
+====
+
+
+.Procedure: Enable and Synchronize PTF repositories using the command line
+
+. On the console enter ``mgr-sync refresh``
+. Enter ``mgr-sync list channel`` and look for channels starting with your SCC account name and `ptfs` in its name. E.g. `a123456-sles-15.3-ptfs-x86_64`
+. Enable the PTF channel with ``mgr-sync add channel <label>``
+
+Now this channel is available and can be added to every system which is using the same base channel.
+
+PTF packages need to be installed explicitly, since they are not automatically picked up when updating a system.
+The SUSE Customer Support will provide the PTF number to fix a specific problem. With the number the proxy package can be identified in the PTF list.
+In {productname} {webui} every system with PTFs available for installation has a page which list them.
+
+
+.Procedure: Installation of PTF packages
+
+. In the {productname} {webui}, navigate to menu:Systems[Systems List] and select the client where you want to install a PTF.
+. Navigate to the menu:Systems[Software > Packages > Software Channels] and select the [systemitem]``PTF channel``.
+. Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
+. To check if the channel assignment is finished, navigate to menu:Systems[Events > History] to see the results.
+. Navigate to the menu:Systems[Software > PTFs > Install] sub tab
+. Select the ptf package you want to install
+. Click btn:[Install PTF], and [guimenu]``Confirm Program Temporary Fixes (PTFs) Installation`` with btn:[Confirm].
+. To see PTF installation results, navigate to menu:Systems[Events > History].
+
+
+In case a PTF should be installed using the API, the normal [systemitem]``system.schedulePackageInstall`` api can be used with the proxy package name.
```

Comment:
```suggestion
In case a PTF should be installed using the API, the normal [systemitem]``system.schedulePackageInstall`` API can be used with the proxy package name.
```